### PR TITLE
docs: update guide.md

### PR DESCRIPTION
**Pull request by YaseenYo**
I have added a missing statement to make docs more clear and user friendly.

**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

**Review by derberg — COMMENTED** [review]
if we want to be so detailed in dev guide I suggest that in the part about forking and cloning we explain how to configure fork locally using https://github.com/asyncapi/.github/blob/master/git-workflow.md instruction

**Comment by magicmatatjahu** [comment]
@YaseenYo We have explained how to contribute to this repo in the [`Contributing`](https://github.com/asyncapi/asyncapi-react/tree/next#contribution) section, so I don't think so that we should go with that direction - also we have [`Development`](https://github.com/asyncapi/asyncapi-react#development) section, so everything is inside Readme.md. Do you wanna add something? I will close it after your comment :)

**Comment by derberg** [comment]
@magicmatatjahu we have nothing there about forking and how to configure fork because there is a custom contrib guide instead of using https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md

**Comment by magicmatatjahu** [comment]
@derberg right, I will change it, or @YaseenYo do you wanna change that link in this PR? Btw. we have still info about forking in this custom guide 😅

**Comment by YaseenYo** [comment]
@magicmatatjahu its fine, you can change it

File: docs/development/guide.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ If you make any changes in the project structure, remember to update it.
 
 ## Install dependencies
 
+Fork the repository and clone it to your development editor. Then follow the below steps.
```
Comment (fmvilas) [inline docs/development/guide.md:42]: Just a suggestion:

```suggestion
Fork the repository and clone it to your development editor. Then follow the steps below.
```

**Pull request closed without merging**